cli/root.go

```diff
@@ -107,9 +108,32 @@ func Root() *cobra.Command {
 	cmd.PersistentFlags().Bool(varNoOpen, false, "Block automatically opening URLs in the browser.")
 	_ = cmd.PersistentFlags().MarkHidden(varNoOpen)
 
+	// Cobra automatically uses "-v" and "--version" for printing the version on
+	// the root cmd. If we decide to use "-v" as a verbose flag on the root,
+	// then that will be a behavior change. So we should just reserve "-v"
+	// and make users use "coder --version" or "coder version"
+	cmd.Flags().BoolP("placeholder", "v", false, "This flag is a placeholder to make the cobra version flag work appropriately")
+	_ = cmd.Flags().MarkHidden("placeholder")
```

Comment:
Cobra doesn't add this flag if `Version` isn't specified in the root. I'd propose that being a cleaner alternative.

Having `--version` and `coder version` seems unnecessary to me. I'd prefer to have `coder version`.

Reply:
I’m fine with only having `version` but having both is consistent with `help` (or do I misremember?) and I think covers a common use case. I.e. there’s a 50/50 chance someone will blindly do either `--version` or `version` instead of checking help 😄.

Reply:
I like how Go just does `go version`. I haven't found it to be confusing tbh

Reply:
I'll drop the flag for now then and just do `coder version`